glossary/access-token.md

```diff
@@ -4,15 +4,13 @@ sublayout: glossary
 title: Access Tokens
 ---
 
+### NOTE: Access tokens are currently **deprecated** and should not be used. [Robot Accounts](/glossary/robot-accounts.html) are their replacement.
```

Comment:
idk if this should be an h3, it's important, yes, but headers are for titles of content sections and this is a whole sentence.